tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/JavaScript.cs

```diff
@@ -140,6 +142,11 @@ public class JavaScript : LanguageBase
 
             var projectDirectory = Path.Combine(WorkingDirectory, project);
 
+            if(profile){
+                var stripPackageName = primaryPackage.Split(new char[] { '/' })[1]; // "@azure/storage-blob" -> "storage-blob"
```

Comment:
```suggestion
                var stripPackageName = primaryPackage.Substring(primaryPackage.LastIndexOf('/') + 1); // "@azure/storage-blob" -> "storage-blob"
```

This code is slightly better, because it also handles packages without a namespace (e.g. `mocha`).  `"mocha".LastIndexOf('/')` will return `-1`, so this will keep the whole name.

Reply:
Got this from stackoverflow 😋

Updated